README.md

```diff
@@ -72,6 +71,46 @@ def mock_server(request):
 This will take care of starting and stopping the Mountebank server for you. Examples of more complex predicates can be 
 found in the [integration tests](https://github.com/brunns/mbtest/tree/master/tests/integration/).
 
+## Use with Docker
+
+If you want to use your own mountebank service instance ([Docker](https://hub.docker.com/r/andyrbell/mountebank), for example) you have **no need to use npm** requirements.
+```sh
+docker run -p 2525:2525 -p IMPOSTER_PORT:IMPOSTER_PORT -d andyrbell/mountebank
```

Comment:
Any reason we're not using [Brandon's docker image](https://hub.docker.com/r/bbyars/mountebank)? That's as close to being the official version as we are likely to get, and is most likely to be kept up to date.

Reply:
andyrbell/mountebank's image size = 23.69 MB
bbyars/mountebank's image size = 345.23 MB